Rakefile

```diff
@@ -19,6 +21,28 @@ def run_command(command)
   end
 end
 
+def parse_release_tag(tag)
+  regex = /^v?(\d+)\.(\d+)\.(\d+).*$/
```

Comment:
Might be an idea to allow an upper case V at the start.  I have made releases with one by mistake in the past.